routes/index.js

```diff
@@ -0,0 +1,89 @@
+import express from 'express';
+import Thought from '../models/thought';
+import { celebrate, Segments, Joi } from 'celebrate';
+Joi.objectId = require('joi-objectid')(Joi);
+
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
+  const thoughts = await Thought.find()
+    .sort({ createdAt: 'desc' })
+    .limit(20)
+    .exec();
+
+  if (thoughts) {
+    res.json(thoughts);
+  } else {
+    res.status(404).json({
+      message: `No thoughts found`,
+      error: err.errors
+    });
+  }
+});
+
+router.post(
+  '/',
+  celebrate({
+    [Segments.HEADERS]: Joi.object({
+      'content-type': Joi.string()
+        .required()
+        .valid('application/json')
+    }).unknown(),
+    [Segments.BODY]: Joi.object()
+      .keys({
+        message: Joi.string()
+          .min(5)
+          .max(140)
+          .trim()
+          .required()
```

Comment:
Interesting, but this is actually a bit problematic 
![image](https://user-images.githubusercontent.com/807352/74823597-3fe3ed80-5307-11ea-84fe-22396aec8fca.png)